plugins/pmsmart/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout charset="UTF-8" noConsoleNoAnsi="true"
+                pattern="%date{HH:mm:ss.SSS} [%thread] %highlight{%-5level} %logger{1.} - %message%n" />
+        </Console>
+        <RollingFile name="File" fileName="${sys:user.home}/.whiterabbit/logs/white-rabbit.log"
```

Comment:
Do you log to the file by intention? Please remove the RollingFile appender if possible.

Reply:
done